docker-compose.yml

```diff
@@ -89,9 +89,56 @@ services:
     env_file:
       - .env
     environment:
-      CELERY_CMD: celery beat --app=geonode.celery_app:app -l INFO -f /var/log/celery.log
+      CELERY_CMD: celery worker -A geonode.celery_app -c 4 -P threads -l INFO -f /var/log/celery.log
       IS_CELERY: 'True'
 
+  celerybeat:
+    restart: unless-stopped
+    build: .
+    container_name: celerybeat4${COMPOSE_PROJECT_NAME}
+    stdin_open: true
+    # tty: true
+    labels:
+      org.geonode.component: celerybeat
+      org.geonode.instance.name: geonode
+    depends_on:
+      - db
+      - rabbitmq
+      - data-dir-conf
+    volumes:
+      - statics:/mnt/volumes/statics
+      - geoserver-data-dir:/geoserver_data/data
+      - backup-restore:/backup_restore
+    env_file:
+      - .env
+    environment:
+      CELERY_CMD: celery beat -A geonode.celery_app -l INFO -f /var/log/celery.log
+      IS_CELERY: 'True'
+
+  # - Uncomment to enable Flower celery monitor
+  # celerymonitor:
+  #   restart: unless-stopped
+  #   build: .
+  #   container_name: celerymonitor4${COMPOSE_PROJECT_NAME}
+  #   stdin_open: true
+  #   # tty: true
+  #   labels:
+  #     org.geonode.component: celerymonitor
+  #     org.geonode.instance.name: geonode
+  #   depends_on:
+  #     - db
+  #     - rabbitmq
+  #     - data-dir-conf
```

Comment:
Thanks @afabiani why does the service depend on data-dir-conf and needs access to backup_restore? Will it catch the tasks emitted by GeoServer as well?

Reply:
Nope, just left due to copy and paste. It can be removed, but at the same time it does not impact the service startup.

Reply:
then I would vote to remove it. Otherwise one might wonder what it's for ;)